mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -174,6 +183,35 @@ void registerGroupShouldCreateRetryExchange() throws Exception {
     class ConcurrentTest implements EventBusConcurrentTestContract.MultiEventBusConcurrentContract,
         EventBusConcurrentTestContract.SingleEventBusConcurrentContract {
 
+        @Test
+        void rabbitMQEventBusCannotHandleHugeDispatchingOperations() throws Exception {
```

Comment:
I don't like that: test should demonstrate valid behaviours.

If tests do not pass expected valid behaviour then we should ignore it.

Reply:
I didn't mean to solve the issue. I just want to show to people that issue.

Reply:
Shouldn't you add the `POC` label to this PR then?

Reply:
Please rename that test in `dispatchShouldHandleLoadBulksGracefully`